packages/eslint-config/index.cjs

```diff
@@ -6,7 +6,6 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:@typescript-eslint/strict',
     'plugin:jsdoc/recommended-typescript-error',
-    'plugin:prettier/recommended',
```

Comment:
Out of curiosity, why was this removed?

Reply:
Basically this plugin runs `prettier` in `ESLint` but as of #2764, we won't need to do that anymore. It should also speed up `ESLint` a bit.